bot.py

```diff
@@ -3,14 +3,15 @@
 from telegram.ext import Updater, MessageHandler, CommandHandler, Filters
 
 from commands.links import links
+from commands.issues import issues
 from commands.help import help
 from commands.xkcd import xkcd
 from commands.welcome import welcome
 from commands.meetup import set_meetup, meetup
 from commands.tweepyStreaming import tweets
 
-bot_name = os.getenv("bot_name")
-token = os.getenv("token")
+bot_name = os.getenv('bot_name', 'OnyxBotTest')
+token = os.getenv('token', '1987248424:AAHJOY0Sw8qRDlYQp4gJfmX4gB21IzH0XF8')
```

Comment:
Please remove this bot username and its token.

Reply:
@Onyx2406 Change it to
```
bot_name = os.getenv('bot_name')
token = os.getenv('token')
```

Reply:
done